app/build.gradle

```diff
@@ -5,11 +5,11 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 
 android {
-    compileSdkVersion 26
+    compileSdkVersion 29
```

Comment:
Now AndroidX is used by default, there are some incorrect import statements in class files such as `DemoViewModel` would you please update them so that the app runs correctly?